modules/engines/engine-xsjs/pom.xml

```diff
@@ -166,6 +166,7 @@
   </dependencies>
 
   <properties>
+    <maven.clean.plugin.directory>target</maven.clean.plugin.directory>
```

Comment:
I think the issue here is being more on why there is a difference between the two contents. The should be the same. By default the flow should be to first update the XSK api in the dirigiblelabs/xsk-api and after that execute `mnv clean install -P content` in XSK in order to sync the content. This should be the behaviour we target.